src/mococrw/mac.h

```diff
@@ -145,4 +145,80 @@ class HMAC : public MessageAuthenticationCode {
     std::unique_ptr<Impl> _impl;
 };
 
+class CMAC : public MessageAuthenticationCode {
+public:
+    /**
+     * @brief Constructor
+     * @param hashFunction The hash function which shall be used
```

Comment:
```suggestion
     * @param cipherType The type of encryption function which shall be used
```

Reply:
Nice! I have just saw that the CI failed because of the undocumented parameter :)